evap/staff/importers/json.py

```diff
@@ -0,0 +1,325 @@
+            Course,
+            semester=self.semester,
+            cms_id=data["gguid"],
+            defaults={"name_de": data["title"], "name_en": data["title_en"], "type": course_type},
+        )
+        course: Course
+        course.degrees.set(degrees)
+        course.responsibles.set(responsibles)
+
+        if changes:
+            self.statistics.updated_courses.append(course)
+        if created:
+            self.statistics.new_courses.append(course)
+
+        self.course_map[data["gguid"]] = course
+
+        return course
+
+    # pylint: disable=too-many-locals
+    def _import_evaluation(self, course: Course, data: ImportEvent) -> Evaluation:
+        course_end = datetime.strptime(data["appointments"][0]["end"], self.DATETIME_FORMAT)
+
+        if data["isexam"]:
+            # Set evaluation time frame of three days for exam evaluations:
+            evaluation_start_datetime = course_end.replace(hour=8, minute=0) + timedelta(days=1)
+            evaluation_end_date = (course_end + timedelta(days=3)).date()
+
+            name_de = "Klausur"
+            name_en = "Exam"
+        else:
+            # Set evaluation time frame of two weeks for normal evaluations:
+            # Start datetime is at 8:00 am on the monday in the week before the event ends
+            evaluation_start_datetime = course_end.replace(hour=8, minute=0) - timedelta(
+                weeks=1, days=course_end.weekday()
+            )
+            # End date is on the sunday in the week the event ends
+            evaluation_end_date = (course_end + timedelta(days=6 - course_end.weekday())).date()
+
+            name_de, name_en = "", ""
+
+        # If events are graded for any degree, wait for grade upload before publishing
+        wait_for_grade_upload_before_publishing = any(filter(lambda grade: grade["scale"], data["courses"]))
+
+        participants = self._get_user_profiles(data["students"])
+
+        defaults = {
+            "name_de": name_de,
+            "name_en": name_en,
+            "vote_start_datetime": evaluation_start_datetime,
+            "vote_end_date": evaluation_end_date,
+            "wait_for_grade_upload_before_publishing": wait_for_grade_upload_before_publishing,
+        }
+        evaluation, created = Evaluation.objects.get_or_create(
+            course=course,
+            cms_id=data["gguid"],
+            defaults=defaults,
+        )
+        if evaluation.state < Evaluation.State.APPROVED:
+            direct_changes = update_with_changes(evaluation, defaults)
+
+            participant_changes = set(evaluation.participants.all()) != set(participants)
+            evaluation.participants.set(participants)
+
+            lecturers_changes = False
+            for lecturer in data["lecturers"]:
+                __, lecturer_created, lecturer_changes = self._import_contribution(evaluation, lecturer)
+                if lecturer_changes or lecturer_created:
```

Comment:
Can we maybe rename `lecturers_changes` and/or `lecturer_changes` to be more distinguishable?